thorlcr/thorutil/thormisc.cpp

```diff
@@ -762,6 +762,17 @@ void loadCmdProp(IPropertyTree *tree, const char *cmdProp)
 
 const LogMsgJobInfo thorJob(UnknownJob, UnknownUser); // may be improved later
 
+static ILogMsgHandler *logHandler = nullptr;
+ILogMsgHandler *queryLogHandler()
```

Comment:
This doesn't appear to be used.

Reply:
@ghalliday - That's true, it isn't in this PR.
But was in my testing where I used printAllStacks()

It's useful in order to get the file handle from the log handler and the redirect then pass it to printAllStacks() , which will then redirect the output to the log.